### ISSUE TITLE
[BLOCKER] 1.1.6 is incompatible with RxJava 2

### ISSUE THREAD

**Issue by earring** [Scorocode/scorocode-SDK-java#2]
Version 1.1.6 of library is incompatible with RxJava 2. Build is failed with error
```java
Error:Execution failed for task ':app:transformResourcesWithMergeJavaResForDebug'.
> com.android.build.api.transform.TransformException: com.android.builder.packaging.DuplicateFileException: Duplicate files copied in APK META-INF/rxjava.properties
  	File1: /home/artem/.gradle/caches/modules-2/files-2.1/io.reactivex/rxjava/1.1.5/ece7b5d0870e66d8226dab6dcf47a2b12afff061/rxjava-1.1.5.jar
  	File2: /home/artem/.gradle/caches/modules-2/files-2.1/io.reactivex.rxjava2/rxjava/2.1.0/2fdf84dedcaaeabb9d70cde9dbb8aad4eccb80a1/rxjava-2.1.0.jar
```
Corresponding snippet from build.gradle
```java
dependencies {
    ...
    compile 'ru.prof-itgroup:scorocode_sdk:1.1.6'
    ...
    compile 'io.reactivex.rxjava2:rxandroid:2.0.1'
    compile 'io.reactivex.rxjava2:rxjava:2.1.0'
}
```

**Comment by PeterStaranchuk** [comment]
Unfortunately, this is RxJava issue.
Please, read this conversation:
https://github.com/ReactiveX/RxJava/issues/4445

this code in gradle file have to fix your problem:

packagingOptions {
exclude 'META-INF/rxjava.properties'
}

**Comment by PeterStaranchuk** [comment]
If you will not be able to fix your issue with this information - please send message here.

**Comment by PeterStaranchuk** [comment]
rxjava issue fixed at our side at version 1.1.7
